opta/cli.py

```diff
@@ -61,6 +63,7 @@ def cli() -> None:
         # after the command.
         # However, we should still clean them up before the next command, or
         # else it may interfere with it.
+        meister.MEISTER_REPORT_ENABLED = VERSION == DEV_VERSION
```

Comment:
Might be better to have a separate flag for this

Reply:
Or an env var